website/content/docs/ui/custom-messages/index.mdx

```diff
@@ -0,0 +1,101 @@
+---
+layout: docs
+page_title: Manage custom messages
+description: >-
+  Use custom messages in the Vault UI to share system-wide alerts.
+---
+
+# Manage custom messages in the Vault UI
+
+Use custom banners and modals in the Vault UI to share system-wide alerts for all Vault UI users.
+
+<Tip title="Best practices for UI messages">
+
+1. **Messages are sticky**. Users can only dismiss messages **temporarily**.
+   The message reappears if the user refreshes their browser window or logs out
+   of the current session.
+
+1. **Messages are intrusive**. Limit the number of active messages to minimize
+   the intrusion on users and reduce the chances that they will dismiss the
+   message without reading it.
+
+1. **Messages are inheritable**. Child namespaces inherit all messages created
+   on the parent namespace. Take advantage of inheritance to reach the greatest
+   number of users with the smallest number of messages.
+
+1. **Delete old messages**. Vault supports a maximum of 100 messages at a
+   time. Practice good message hygiene by regularly deleting expired and outdated
+   messages.
+
+</Tip>
+
+## Before you start
+
+- **You must have Vault 1.16.0 or higher installed**.
+- **You must have permission to call the `sys/config/ui/custom-message` endpoint**:
+  - To **create or edit messages** you must have `read`, `list`, and `create` permissions.
+  - To **delete messages** you must have `list` and `delete` permissions.
+
+## Add a custom message
+
+1. Navigate to the **Settings** section in the Vault UI sidebar and select "Custom
+   Messages".
+
+1. On the **Custom messages** page, select whether you want the message
```

Comment:
should these numbers be increasing?

Reply:
The standard convention for markdown is to keep it at `1.` and it will auto format those numbered bullets to be increasing :)

Reply:
@Monkeychip This complies with the [markdown style guide](https://github.com/hashicorp/engineering-docs/blob/main/writing/markdown.md) as-is, so this is fine.